crates/accelerate/src/quantum_circuit/circuit_data.rs

```diff
@@ -262,8 +269,9 @@ impl CircuitData {
             .try_insert(BitAsKey::new(bit)?, idx)
             .is_ok()
         {
+            let py = bit.py();
```

Comment:
I'm not sure we have any formally establish pattern with the PyO3 token, but perhaps for consistency it'd be better to keep it as an explicit argument to any public function which uses it.

Reply:
Reverted in 5048d15389.